### PR TITLE
Minor front end changes for KPI cards and Child Beneficiary List

### PR DESCRIPTION
**Pull request by awalkowiak**
https://manage.dimagi.com/default.asp?265770
https://manage.dimagi.com/default.asp?266715
https://manage.dimagi.com/default.asp?266716
https://manage.dimagi.com/default.asp?266721

@lwyszomi @emord

### CODE REVIEW

File: custom/icds_reports/static/js/directives/additional-filter/additional-filter.directive.js [hunk 1]
```diff
@@ -2,9 +2,9 @@ function AdditionalModalController($location, $uibModalInstance, filters, gender
     var vm = this;
     vm.filters = filters;
 
-    vm.genders = genders;
+    vm.genders = genders.slice();
```
Comment (lwyszomi) [inline custom/icds_reports/static/js/directives/additional-filter/additional-filter.directive.js]: Could you use ```window.angular.copy(genders)```

File: custom/icds_reports/static/js/directives/additional-filter/additional-filter.directive.js [hunk 2]
```diff
@@ -2,9 +2,9 @@ function AdditionalModalController($location, $uibModalInstance, filters, gender
...
 
-    vm.ages = ages;
+    vm.ages = ages.slice();
```
Comment (lwyszomi) [inline custom/icds_reports/static/js/directives/additional-filter/additional-filter.directive.js]: Could you use ```window.angular.copy(ages)```

**Review by lwyszomi — CHANGES_REQUESTED** [review]
Hi,

I added two comments.

**Comment by awalkowiak** [comment]
@lwyszomi I made requested change.

**Pull request merged**